README.md

```diff
@@ -212,6 +212,656 @@ struct Person2 {
 }
 ```
 
+## Reference
+The following table provides a range of examples showing how to declare data types with `rasn`.
+
+<table>
+<tr>
+<td></td> <td> ASN1 </td> <td> rasn </td>
+</tr>
+<tr>
+<td>Type alias</td>
+<td>
+
+```asn
+Test-type-b ::= BOOLEAN
+Test-type-a ::= Test-type-b
+```
+
+</td>
+<td>
+
+```rust
+# use rasn::prelude::*;
+
+#[derive(AsnType, Decode, Encode)]
+#[rasn(delegate)]
+struct TestTypeB(pub bool);
+
+/// either
+type TestTypeA = TestTypeB;
+
+/// or
+#[derive(AsnType, Decode, Encode)]
+#[rasn(delegate)]
+struct TestTypeA2(pub TestTypeB);
```

Comment:
One thing I'm not sure about here, is that you don't mention that just a type alias is sufficient. You don't need to have delegate types if there's no restrictions or you don't want/need to implement custom methods for it.

```rust
type TestTypeB = bool;
```

Reply:
True. I'll also add the possibility of type aliases in the other examples.